src/app.ts

```diff
@@ -179,33 +188,40 @@ export async function newApp(
   // Setup routes
   //
 
-  await registerLoginRoute(app);
-  registerPublicRoutes(app);
-  registerAuthenticationRoutes(app, authenticationBasePath);
-  registerAPIRoutes(app, APIBasePath, allowNotifyIPSourceRange);
-  registerPagoPARoutes(app, PagoPABasePath, allowPagoPAIPSourceRange);
-
-  const idpMetadataRefreshIntervalMillis =
-    container.resolve<number>(IDP_METADATA_REFRESH_INTERVAL_SECONDS) * 1000;
-  const idpMetadataRefreshTimer = startIdpMetadataUpdater(
-    app,
-    idpMetadataRefreshIntervalMillis
-  );
-  app.on("server:stop", () => {
-    clearInterval(idpMetadataRefreshTimer);
-  });
+  try {
+    const newSpidStrategy = await defaultModule.loadSpidStrategy();
```

Comment:
why is this `newSpidStrategy` and not just `spidStrategy`

Reply:
because now spidStrategy is used for the import in [this](https://github.com/teamdigitale/io-backend/pull/436/files#diff-f98485da1849ae631ba5c04e5cec07ddR28) row